geometry/demesh.js

```diff
@@ -0,0 +1,19 @@
+import {
+  deletePendingSurfaceMeshes,
+  demesh as demeshWithCgal,
+} from '@jsxcad/algorithm-cgal';
+
+import { linearize } from './tagged/linearize.js';
+import { replacer } from './tagged/visit.js';
+import { toConcreteGeometry } from './tagged/toConcreteGeometry.js';
+
+const filter = (geometry) => ['graph'].includes(geometry.type);
+
+export const demesh = (geometry) => {
```

Comment:
I thought this was added a while back? It's the opposite of remesh, right?

Reply:
Yeah, it's replacing geometry/tagged/demesh.js